docs/src101/src101.md

```diff
@@ -0,0 +1,179 @@
+# SRC-101 Token Standard
+
+`SRC-101` is a Non-fungible Token Protocol on Bitcoin Stamps. It's almost backward compatible with SRC-20, but has new features and has no risk of loss assets by mistakenly spending them with UTXOs. It greatly supports high-quality pictures and complex metadata. It's arming to give Non-fungible tokens more possibilities and scalability.
+
+# Specifications
+SRC-101 Tokens must conform to these **required** fields (without **optional** label) or a Bitcoin Stamp Number will not be created, the transaction will not be considered a valid SRC-101 transaction, and they will not appear in the Bitcoin Stamps Protocol index / API. 
+
+SRC-101 transaction must be signed and broadcast onto BTC by the address that holds the SRC-101 token balance as it acts as a means to authenticate ownership. Both the source and destination addresses are embedded into the BTC transaction which is created by the users wallet. The SRC-101 reference wallet will ensure you are creating the proper transaction until support can be integrated into more broadly distributed wallets such as Hiro. Please use extreme caution if signing transactions created by a third party.
+
+### DEPLOY
+```JSON
+{
+    "p": "src-101", //(string)protocol standard for non-fungible token
+    "op": "deploy", //(string)function name
+    "tick": "Stamp Is Cool",  //(string)collection name, must be 1-32 characters, can have blank.
+    "max": "10000", //(uint64)maximum number of mints, if null means unlimited.
+    "lim": "5", //(uint64)limit per mint. If null means unlimited.
+    "dec": "0", //(uint8)decmial, current must be 0, src-20 compatible, maybe support divisible NFT to FT in the future
+    "owner": "bc1q34eaj4rz9yxupzxwza2epvt3qv2nvcc0ujqqpl", //(string)token owner address who can invoke operation methods, support any type of bitcoin addresses
+    "rec": "bc1q7rwd4cgdvcmrxm27xfy6504jwkllge3dda04ww", //(string)recipient address to receive mint fees, support any type of bitcoin addresses
```

Comment:
When do we receive mint fees, and what is the difference between it and price.

Reply:
> When do we receive mint fees

In the same mint transaction

> what is the difference between it and price.
For these people not in whitelist, this price = mint price. For these people in whitelist, mint price = disc * price.